src/current/v23.2/encryption.md

```diff
@@ -70,7 +70,12 @@ The report shows encryption status for all stores on the selected node, includin
 - Active data key information.
 - The fraction of files/bytes encrypted using the active data key.
 
-CockroachDB relies on [storage layer]({% link {{ page.version.version }}/architecture/storage-layer.md %}) compactions to write new files using the latest encryption key. It may take several days for all files to be replaced. Some files are only rewritten at startup, and some keep older copies around, requiring multiple restarts. You can force storage compaction with the `cockroach debug compact` command (the node must first be [stopped]({% link {{ page.version.version }}/node-shutdown.md %}#perform-node-shutdown)).
+CockroachDB relies on [storage layer compactions]({% link {{ page.version.version }}/architecture/storage-layer.md %}#compaction) to write new files using the latest encryption key. It may take several days for all files to be replaced. Some files are only rewritten at startup, and some keep older copies around, requiring multiple restarts. You can force storage compaction with the `cockroach debug compact` command (the node must first be [stopped]({% link {{ page.version.version }}/node-shutdown.md %}#perform-node-shutdown)).
+
+The fraction of files/bytes encrypted on the store may be less than 100% for the following reasons:
+
+1. The percentage shown is the percentage encrypted with the _current_ data key, which rotates at the configured [`rotation-period`](#starting-a-node-with-encryption). When a data key rotates, the percentage will drop down to zero and slowly climb up as data is compacted.
```

Comment:
Maybe use an unordered list so as not to imply that these occur together.

Also, I'm not sure it is clear where the percentage is shown, unless this is elsewhere/nearby in the doc?

```suggestion
- The percentage shown is the percentage encrypted with the **current** data key, which rotates at the configured [`rotation-period`](#starting-a-node-with-encryption). When a data key rotates, the percentage will drop down to zero and slowly climb up as data is compacted.
```

Reply:
Added, thanks!

It's mentioned at the beginning of this section 

> Encryption status can be seen on the node's stores report, reachable through: `http(s)://nodeaddress:8080/#/reports/stores/local` (or replace `local` with the node ID). For example, if you are running a [local cluster]({% link {{ page.version.version }}/secure-a-cluster.md %}), you can see the node's stores report at `https://localhost:8080/#/reports/stores/local`.
>
> The report shows encryption status for all stores on the selected node, including:
> 
> - Encryption algorithm.
> - Active store key information.
> - Active data key information.
> - The fraction of files/bytes encrypted using the active data key.
>